packages/wallet/ui/src/components/AppBar.jsx

```diff
@@ -55,6 +58,17 @@ const AppBar = () => {
         </a>
       </div>
       <div className={classes.appBarSection}>
+        <div className={classes.connector}>
+          <Tooltip title="Help">
+            <IconButton
+              color="primary"
+              size="medium"
+              href="https://agoric.com/documentation/guides/wallet/ui.html"
```

Comment:
I'd like to see some sort of documentation about this external dependency. An in-line string constant like this suggests it's a purely local design choice. A named constant seems like a useful form of documentation.

Reply:
Made this into a constant, PTAL